package.json

```diff
@@ -28,20 +28,20 @@
   "dependencies": {
     "dasherize": "^2.0.0",
     "hyperhtml": "^2.13.0",
-    "hyperhtml-element": "^2.1.1"
+    "hyperhtml-element": "^2.3.0 <3.0"
```

Comment:
is there a reason why we are not migrating to v3?

Reply:
I did not read about or try v3 yet. So idk if there are any breaking changes.
Also, since some projects are already using this version, I wanted to err on the side of caution.

Reply:
seems there is not much information about whats changed in 3.x :/ i have opened a ticket so we keep in mind to update and test it.